system-tests/dsp-compatibility-tests/compatibility-test-runner/src/test/java/org/eclipse/edc/tck/dsp/EdcCompatibilityTest.java

```diff
@@ -32,57 +33,74 @@
 
 import java.nio.file.Path;
 import java.util.HashMap;
+import java.util.List;
 
+import static org.assertj.core.api.Assertions.assertThat;
 import static org.eclipse.edc.util.io.Ports.getFreePort;
 
-@Disabled(value = "Need to disable this test until the connector is 100% compliant with DSP")
 @NightlyTest
 @Testcontainers
 public class EdcCompatibilityTest {
 
     private static final GenericContainer<?> TCK_CONTAINER = new TckContainer<>("eclipsedataspacetck/dsp-tck-runtime:latest");
+    @RegisterExtension
+    protected static RuntimeExtension runtime = new RuntimePerClassExtension(new EmbeddedRuntime("CUnT",
```

Comment:
maybe it has a particular meaning, but the embedded runtime looks... un-polite :)